paper/implementation.tex

```diff
@@ -20,7 +20,42 @@
 
 \subsection{Code Editor}\label{subsec:code-editor}
 
-% todo decribe the code editor component and how it works
+The code editor is a text editor program designed for editing source code of computer programs.
```

Comment:
This doesn't make a lot of sense. Our code editor does not allow editing source code.
```suggestion
The code editor is a text editor program designed for editing the \cfgfiles.
```